### DISCUSSION TITLE
Go to next chat modal window in the chat list

### DISCUSSION THREAD

**Discussion by ludoja** [LiveHelperChat/livehelperchat#1937] [Q&A]
Hi, when reviewing/controlling operators' chats from a list (as a supervisor) in the modal window I have to click on icon (i) of each chat to view it https://bit.ly/3zX5OSu.

Is there any button or shortcut (like CTRL K/CTRL J in gmail) which would enable me to go to next (previous) chat from the list?

It would also help if is a way to resize the modal window, which would enable me to accustom it to screen height - to minimise the vertical scrolling.

Thanx for tip/answer.

**Comment by remdex** [comment]
Use dashboard there is shortcut to switch between chat tabs

**Comment by ludoja** [comment]
Thanx for the reply Rem, but I am not sure whether we understand each other.

Pls see https://bit.ly/3MGyjvg
I have a saved search (list) of chats, (1) and I want to view those chats in the modal window (2) one by one, without need to click the (i) icon by the chat row. 
As the chats are not numbered, it often happens, that I click on the same chat, or skip one, when going though the list, so an action which would enable me to scroll the chat content via modal window would be very helpfull.

I had a look at dashboard option, but it is niot clear to me how it could help me in the task mentioned above.

**Reply by remdex** [reply]
Now it's more clear if wish to sponsor that you can contact me, otherwise I have just to many paid support 😀
